autoload/neomake/makers/ft/scala.vim

```diff
@@ -1,11 +1,23 @@
 " vim: ts=4 sw=4 et
 function! neomake#makers#ft#scala#EnabledMakers()
-    let makers = ['scalac']
-    if neomake#utils#Exists('scalastyle')
+    let makers = get(g:, 'neomake_scala_fast', 0) ? ['fsc'] : ['scalac']
+    if neomake#utils#Exists('scalastyle') && get(g:, 'neomake_scala_style', 1)
```

Comment:
Currently disabling scalastyle is already possible by setting your enabled makers manually:

```
let g:neomake_scala_enabled_makers = ['scalac']
```

I prefer this method over adding another configuration variable, which must then be documented. I already think there's too many configuration variables and it makes the docs a little hard to read through quickly.